README.md

```diff
@@ -9,10 +9,10 @@ NAME:
 USAGE:
    
   ccboc --teff=10000 --logG=4.0 create (Creates a calculation with teff=10000 and LogG=4.0)
-  ccboc get calculation calc-1881i9dh5zvnllip (Gets the calculation with id='calc-1881i9dh5zvnllip')
+  ccboc get calculation --calc-id=calc-1881i9dh5zvnllip (Gets the calculation with id='calc-1881i9dh5zvnllip')
```

Comment:
This is basically the calculation name, not the id. There is no need to give a flag for it. `ccboc get calculation calc-name` is the logical command input.

Reply:
There was an uncomfortable solution beforehand. calcID had to be last argv, now It can be placed anywhere.

Reply:
Yea this is what we want. You can remove the entry in the README as well.